connector/mqtt/vernemq/broker/src/dojot_disconnect_plugin/src/utils.erl

```diff
@@ -0,0 +1,42 @@
+-module(utils).
+-import(lists,[member/2]).
+
+% time defined in miliseconds (default 30 min)
+-define(MAX_TIMEOUT, element(1, string:to_integer(os:getenv("PLUGIN_DISC_LIFETIME_SESSION", "1800000")))).
+-define(DISCONNECT_ETS_TABLE, disconnect_ets_table).
+
+-export([
+    set_connection_timeout/1,
+    disconnect_client/1,
+    is_dojot_user/1,
+    cancel_connection_timeout/1
+]).
+
+is_dojot_user(Username) ->
+
+    % we will split the username. If dojot username is tenant:deviceid
+    % the length of list will always 2. If not, it will not be dojot user.
+    Res = binary:split(Username, <<":">>, [global]),
+    case length(Res) of
+        2 ->
+            is_user;
+        _->
+            not_user
+    end.
+
+disconnect_client(SubId) ->
```

Comment:
Shouldn't this method remove the entry from ETS too, otherwise, the client will be disconnected, but the entry will be kept. This can be a problem if clients reconnect using different Client IDs or never reconnect again.